mappings/net/minecraft/item/FoodItemSetting.mapping

```diff
@@ -0,0 +1,37 @@
+CLASS awp net/minecraft/item/FoodItemSetting
+	CLASS awp$a Builder
+		FIELD a hunger I
+		FIELD b saturation F
+		FIELD c wolfFood Z
```

Comment:
Is this specifically wolf food or could it be considered to be more generally 'meat'?

Reply:
We called it "wolfFood" on the old FoodItem, and its only use in vanilla is whether wolves can eat it. I don't remember if cooked fishes count as wolf food, but then again whether fish is considered a type of meat is already super vague culinarily.

Reply:
Wolves can only be fed chicken, pork, beef, mutton, rabbit and rotten flesh according to the [Minecraft Wiki](https://minecraft.gamepedia.com/Wolf#Taming.2C_health_and_feeding)

Reply:
Some background on "is fish meat": various branches of Christianity have at various times considered fish as non-meat, so that it could be eaten during lent to prevent starvation. there's also a distinction between "meat" and "fish" in cooking due to the vastly different types of flavors domesticated animal meat and fish can have, but there's also a distinction between "meat" and "game meat", or hunted animal meat, for the same reason. However, fish is still structurally a type of meat, so I don't feel "is this meat" is a good descriptor for its usage in Vanilla.